tools/public_api_guard/material/list-testing.md

```diff
@@ -87,7 +87,6 @@ export class MatListOptionHarness extends MatListItemHarnessBase {
     focus(): Promise<void>;
     getCheckboxPosition(): Promise<MatListOptionCheckboxPosition>;
     static hostSelector: string;
-    isDisabled(): Promise<boolean>;
```

Comment:
Surprised that `MatListItemHarnessBase` does not show up in the api golden, good thing to note.

Reply:
It's because we only export the derived classes, not the base class.

Reply:
Yeah, I was thinking Microsoft's API extractor would pull in this type regardless, since it's defining a public export signature.